src/internal/packager/pkgcfg/load.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package pkgcfg loads and applies schema migrations to zarf.yaml files.
+package pkgcfg
```

Comment:
nit: does this need to be nested in `packager`? They both work on v1alpha1.ZarfPackage but the behavior isn't necessarily _only_ useful for packager operations.